Sources/MapboxCoreNavigation/IncidentsOptions.swift

```diff
@@ -0,0 +1,18 @@
+
+import Foundation
+
+/**
+ Electronic Horizon supports live incidents on a most probable path. To enable live incidents `IncidentsOptions` should be provided to `NavigationSettings` before starting navigation. If both `graph` and `apiUrl` are empty, live incidents are disabled (by default).
+ */
+public struct IncidentsOptions {
+    /**
+     Incidents provider graph name.
+     */
+    public var graph: String
+    /**
+     LTS incidents service API url.
+     
+     If empty line is supplied will use a default url.
+     */
+    public var apiURL: URL
```

Comment:
You cannot create URL with empty line, `URL(string: "")` will return nil.
Let's just use String, this is the parameter NN accepts

Reply:
Wanted to use URL object for all of it's utility. So probably should allow this one to be nullable instead to maintain functionality...

Reply:
Maybe for parity with Android let's leave it as `String`?
https://docs.mapbox.com/android/navigation/api/2.7.0/libnavigation-base/com.mapbox.navigation.base.options/-incidents-options/

Reply:
We are using `URL` across our SDK for such data. So IMHO it is OK to not mirror Android here